samples/TodoList/android/app/build.gradle

```diff
@@ -127,7 +127,7 @@ android {
 
 dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
+    compile "com.android.support:appcompat-v7:27.1.1"
```

Comment:
Why not current, e.g. v28, or even androidx?

Reply:
He synced the versions with other samples. I support him here as it is easier to install a single set of the dependencies to run all of the samples.
We probably should load the versions of tools and libraries from the properties file instead of defining them in every sample.